src/main/java/mate/academy/bookshop/service/impl/BookServiceImpl.java

```diff
@@ -30,6 +30,11 @@ public BookDto findById(Long id) {
         return bookMapper.toDto(book);
     }
 
+    @Override
+    public void deleteById(Long id) {
+        bookRepository.deleteById(id);

```

Comment:
we should first check if the book with such an `id` exists, if not - throw `EntityNotFoundException`, same for update method